src/styles/blocks/card.scss

```diff
@@ -0,0 +1,135 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  color: black;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  padding: 32px 16px 14px;
+  margin-bottom: 48px;
+
+  &:not(:nth-child(4n)) {
+    margin-right: 48px;
+  }
+
+  &__img {
+    width: 160px;
+    padding-bottom: 40px;
+    padding-left: 3px;
+
+    &-size {
+      display: flex;
+      width: 160px;
+      height: 134px;
+    }
+  }
+
+  &__header {
+    font-weight: 500;
+    line-height: 18px;
+    font-style: normal;
+    font-size: 12px;
+    padding-bottom: 4px;
+    color: $black-link;
+
+    &:link {
+      color: $black-link;
+    }
+
+    &:visited {
+      color: $black-link;
+    }
```

Comment:
```suggestion
    &:link,
    &:visited {
      color: $black-link;
    }
```

Reply:
@RuslanLiapin 
What about this recommendation? Do you agree with it?

Reply:
is not fixed